### PR TITLE
Adjusting documentation based on API behavior.

### PR DESCRIPTION
**Pull request by georgespatton**
We have observed both from the API and from the Azure portal that we can add multiple VMSS to a single AppGW v2.  This change documents this observed behavior.  

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by fengzhou-msft — APPROVED** [review]

**Review by hyonholee — APPROVED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json [hunk 1]
```diff
@@ -8684,7 +8684,7 @@
           "items": {
             "$ref": "#/definitions/SubResource"
           },
-          "description": "Specifies an array of references to backend address pools of application gateways. A scale set can reference backend address pools of multiple application gateways. Multiple scale sets cannot use the same application gateway."
+          "description": "Specifies an array of references to backend address pools of application gateways. A scale set can reference backend address pools of multiple application gateways. Multiple scale sets can use the same application gateway."
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Multiple scale sets still cannot use the loadBalancerBackendAddressPools and loadBalancerInboundNatPools, right?

Comment (hyonholee) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Can you also update compute.json in the previous versions?  (2018-10-01/compute.json, etc)

Comment (georgespatton) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: I didn't test these specifically, this change is only related to application gateways (I tested on AppGWv2).

**Comment by fengzhou-msft** [comment]
@hyonholee @georgespatton any changes need for this PR? Can we merge it?

**Comment by fengzhou-msft** [comment]
any update?

**Comment by fengzhou-msft** [comment]
Close idle PR. Please feel free to reopen it and continue the work if you still need it.

**Pull request closed without merging**